package.json

```diff
@@ -9,7 +9,9 @@
     "linter": "npx eslint .",
     "test-unit": "mocha --require ts-node/register test/unit/**/*.ts",
     "test-integration": "mocha --require ts-node/register test/integration/*.ts",
-    "test-ui": "mocha --timeout 10000 --require ts-node/register test/ui/*.ts"
+    "test-ui": "mocha --timeout 10000 --require ts-node/register test/ui/*.ts",
+    "test-accessibility": "URL_ENV=FRONTEND_URL pa11y-ci --config .pa11yci.js",
+    "test-accessibility-local": "URL_ENV=Local pa11y-ci --config .pa11yci.js"
```

Comment:
I think you've actually gone a little too far, URL_ENV is not needed, and I don't want two commands to run the tests.